scripts/lib/libvirt/libvirt_setup.py

```diff
@@ -87,7 +87,7 @@ def net_config(args):
     return get_config(values, fin)
 
 
-def compute_config(args, cpu_flags=cpuflags()):
+def compute_config(args, cpu_flags=cpuflags(), machine="pc-0.14"):
```

Comment:
Use `machine=None` in the method signature, and this string (declares as `DEFAULT_MACHINE = 'pc-0.14'` at the top of the file, to signalize that is some kind of constant, where also we can add some documentation about why this string) is assgined inside the code:

```python
if not machine:
    machine = DEFAULT_MACHINE
```